plugins/PostProcessingPlugin/scripts/TweakAtZ.py

```diff
@@ -0,0 +1,981 @@
+                    "description": "New temperature for Extruder 2 (T1).",
+                    "unit": "°C  ",
+                    "type": "int",
+                    "default_value": 190,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "160",
+                    "maximum_value_warning": "250",
+                    "enabled": "multi_extruder and f_change_extruder_temperature and taz_enabled"
+                },
+                "g_change_retract": {
+                    "label": "Change Retraction Settings",
+                    "description": "Indicates you would like to modify retraction properties.  If 'Firmware Retraction' is enabled then M207 and M208 lines are added.  Your firmware must understand those commands.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "taz_enabled and not multi_extruder"
+                },
+                "g_change_retract_speed": {
+                    "label": "    Change Retract/Prime Speed",
+                    "description": "Changes the retraction and prime speed.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "g_change_retract and taz_enabled and not multi_extruder"
+                },
+                "g_retract_speed": {
+                    "label": "        Retract/Prime Speed",
+                    "description": "New Retract Feed Rate (mm/s).  If 'Firmware Retraction' is enabled then M207 and M208 are used to change the retract and prime speeds and the distance.  NOTE: the same speed will be used for both retract and prime.",
+                    "unit": "mm/s  ",
+                    "type": "float",
+                    "default_value": 40,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "100",
+                    "enabled": "g_change_retract and g_change_retract_speed and taz_enabled and not multi_extruder"
+                },
+                "g_change_retract_amount": {
+                    "label": "    Change Retraction Amount",
+                    "description": "Changes the retraction length during print",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": "g_change_retract and taz_enabled and not multi_extruder"
+                },
+                "g_retract_amount": {
+                    "label": "        Retract Amount",
+                    "description": "New Retraction Distance (mm).  If firmware retraction is used then M207 and M208 are used to change the retract and prime amount.",
+                    "unit": "mm  ",
+                    "type": "float",
+                    "default_value": 6.5,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "20",
```

Comment:
```suggestion
                    "minimum_value": 0,
                    "maximum_value_warning": 20,
```
Can just use minimum_value.